src/app/views/template.js

```diff
@@ -0,0 +1,39 @@
+import Content from './content';
+
+const Template = (() => {
```

Comment:
This template is fine, but it does not satisfy the requirement of each tab (Contact, Home, Menu) being in it's own module (as seen in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png). The template itself is a module, which is fine, but the content for your tabs should be in their own separate module. 

You will also notice the mention of a Contact and Menu page which Is not clearly implemented/ Does not work as should on your webpage.